BDArmory/Control/MissileFire.cs

```diff
@@ -1939,6 +1943,21 @@ IEnumerator GuardMissileRoutine()
                                     yield return new WaitForFixedUpdate();
                                 }
                             }
+                            if (guardTarget && ml && mlauncher.multiLauncher.turret && vesselRadarData.locked)
```

Comment:
This causes an NRE if `mlauncher.multiLauncher` is `null`.
Also in other places where `.multiLauncher` is accessed: lines 2015, 2076, 2140, 2196, 2226, 2310, 6463.